tls/certificate_store.go

```diff
@@ -47,9 +48,36 @@ func (c CertificateStore) GetAllDomains() []string {
 			allCerts = append(allCerts, domains)
 		}
 	}
+
+	// Get Default certificate
+	if c.DefaultCertificate != nil {
+		allCerts = append(allCerts, c.getCertificateDomains(c.DefaultCertificate)...)
+	}
 	return allCerts
 }
 
+func (c CertificateStore) getCertificateDomains(cert *tls.Certificate) []string {
```

Comment:
can be a function instead of a method:
```go
func getCertificateDomains(cert *tls.Certificate) []string {
	if cert == nil {
		return nil
	}

	x509Cert, err := x509.ParseCertificate(cert.Certificate[0])
	if err != nil {
		return nil
	}

	var names []string

	if len(x509Cert.Subject.CommonName) > 0 {
		names = append(names, x509Cert.Subject.CommonName)
	}

	for _, san := range x509Cert.DNSNames {
		names = append(names, san)
	}

	return names
}
```